apps/reporting/templatetags/extras.py

```diff
@@ -0,0 +1,14 @@
+from datetime import date
+
+import humanize
+from django import template
+
+register = template.Library()
+
+
+def ago(value):
+    delta = humanize.naturaldelta(date.today() - value)
```

Comment:
Is this for more control than provided by https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#timesince?

Reply:
heh, no. I just didn't know about timesince

Reply:
Although, this is a heinous behavior: ```Comparing offset-naive and offset-aware datetimes will return an empty string.```

Reply:
sure. why not error, definitely what the user wanted was for their thing to just disappear

Reply:
although it does make it easy to add `ago` in one place since we use it everywhere

Reply:
Django template filters in general return an empty string if there's an error – reason being they figure a missing string is desirable to a 500.